crates/matrix-sdk-ui/src/room_list_service/state.rs

```diff
@@ -117,7 +117,10 @@ impl Action for AddVisibleRoomsList {
                         SlidingSyncMode::new_selective().add_range(VISIBLE_ROOMS_DEFAULT_RANGE),
                     )
                     .timeline_limit(20)
-                    .required_state(vec![(StateEventType::RoomEncryption, "".to_owned())]),
+                    .required_state(vec![
+                        (StateEventType::RoomEncryption, "".to_owned()),
+                        (StateEventType::RoomMember, "$LAZY".to_owned()),
```

Comment:
If we want, we could also include the current user info by adding another `RoomMember` entry with `$ME` as the value (I've used it in the notification sync, just in case). Would it make sense?

Reply:
Since the `m.room.member` event contains the user's display name, that would be nice to have it. Although I'm not sure if that means the proxy will include it in every single response, or just the first one...

Reply:
I honestly don't know either but seeing as we already have that data is it worth it?